### ISSUE TITLE
Playback for Live Videos doesn't start Immediately

### ISSUE THREAD

**Issue by WINOFFRG** [shaka-project/shaka-player#4654]
**Have you read the [Tutorials](http://shaka-player-demo.appspot.com/docs/api/tutorial-welcome.html)?**
Yes

**Have you read the [FAQ](https://bit.ly/ShakaFAQ) and checked for duplicate open issues?**
Yes

**What version of Shaka Player are you using?**
v4.2.3 (uncompiled)

**Please ask your question**
I have tried few live videos, the behaviour is same on all. As soon as I load/play the video, The first frame is shown and video is paused, Even if I click the play button nothing happens. And the video collects some buffer say 8seconds or 30 seconds on different different manifests. The playback only starts when that specific time has been reached. If the video is off from live time range and `goToLive` is called the same behaviour happens.

Kindly let me know if I should send the Manifest and DRM Config via Mail.

This issue was also raised earlier referernce https://github.com/shaka-project/shaka-player/issues/4559#issuecomment-1279731101

**Comment by WINOFFRG** [comment]
Solve after setting ` manifest.dash.ignoreMinBufferTime` to `true`. Can anyone please drop explanation for this why we have a `MinBufferTime` and why is it set to `false` by default? That would be a great help.

**Comment by theodab** [comment]
`manifest.dash.ignoreMinBufferTime` causes the DASH parser to ignore the `minBufferTime` attribute in the DASH manifest.
We conform to it because it's part of the DASH spec, which is why the configuration value to ignore it is turned off by default.

As to why the minimum buffer time exists to begin with, see the ISO/IEC 23009-1:2022 (downloadable [here](https://standards.iso.org/ittf/PubliclyAvailableStandards/index.html)). Specifically, their description of the `bandwidth` attribute:

> Consider a hypothetical constant bitrate channel of
bandwidth with the value of this attribute in bits per
second (bps). Then, if the Representation is
continuously delivered at this bitrate, starting at any
RAP indicated in this element a client can be assured of
having enough data for continuous playout providing
playout begins after
@minBufferTime * @bandwidth bits have
been received (i.e. at time @minBufferTime after
the first bit is received).

Basically, the minimum buffer time is the minimum wait time to ensure that playback will be smooth. If you don't wait that long, it's possible that (due to the encoder encoding some frames faster than others, stuff like that), the video will need to pause to buffer.

**Comment by WINOFFRG** [comment]
I understand, Thanks a lot
